storybookjs/storybook
# Change extensions ordering in webpack config

## Issue:
Support for mjs was added in https://github.com/storybooks/storybook/pull/4466 but my project revealed some isses with graphql libraries that use mjs.
## What I did
According to hint given in comment: https://github.com/graphql/graphql-js/issues/1272#issuecomment-377384574 I changed ordering of extensions in webpack configuration. After applying this change my project setup was working fine again.

## How to test
Try to tun storybook on project containing libraries that use .mjs extensions - in my case those were graphql related libraries

Is this testable with Jest or Chromatic screenshots?
Probably not
Does this need a new example in the kitchen sink apps?
No
Does this need an update to the documentation?
No

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
